app/uk/gov/hmrc/tradergoodsprofilesrouter/models/response/CreateOrUpdateRecordResponse.scala

```diff
@@ -46,11 +46,11 @@ case class CreateRecordResponse(
   updatedDateTime: String
 )
 
-object CreateRecordResponse {
+object CreateOrUpdateRecordResponse {
 
-  implicit val writes: Reads[CreateRecordResponse] = (json: JsValue) =>
+  implicit val writes: Reads[CreateOrUpdateRecordResponse] = (json: JsValue) =>
     JsSuccess(
-      CreateRecordResponse(
+      CreateOrUpdateRecordResponse(
```

Comment:
if you do not do any thing with the field you do not need to write a writer and reader like that, but you can get all this with one line of code:

implicit val format: OFormat[CreateOrUpdateRecordResponse] = Json.format[CreateOrUpdateRecordResponse]

Reply:
This is set explicitly because of the 22 parameter limit in scala case class

Reply:
forgot about the limit